test/crowdsale/PausableCrowdsale.test.js

```diff
@@ -0,0 +1,52 @@
+const shouldFail = require('../helpers/shouldFail');
+
+const PausableCrowdsale = artifacts.require('PausableCrowdsaleImpl');
+const SimpleToken = artifacts.require('SimpleToken');
+
+require('../helpers/setup');
+
+contract('PausableCrowdsale', function ([_, pauser, wallet, anyone]) {
+  const rate = 1;
+  const value = 1;
+
+  beforeEach(async function () {
+    const from = pauser;
```

Comment:
Not a fan of this tbh, I'd rather each transaction be explicit than having to figure out what `from` means in the current scope.